src/streamsync/core.py

```diff
@@ -983,7 +986,18 @@ def _handle_binding(self, event_type, target_component, instance_path, payload)
             return
         self.evaluator.set_state(binding["stateRef"], instance_path, payload)
 
-    def _call_handler_callable(self, event_type, target_component, instance_path, payload) -> Any:
+    async def _async_handler_executor(self, callable_handler, arg_values):
```

Comment:
I am not an expert on managing async code. I write a proposal that seems to work and allow to keep the code symetry on ``_call_handler_callable``

```python
    def _async_handler_executor(self, callable_handler, arg_values):
        async_callable = self._async_handler_executor_internal(callable_handler, arg_values)
        return asyncio.run(async_callable)

    async def _async_handler_executor_internal(self, callable_handler, arg_values):
        with contextlib.redirect_stdout(io.StringIO()) as f:
            result = await callable_handler(*arg_values)
        captured_stdout = f.getvalue()
        return result, captured_stdout
```